src/sagemaker/base_deserializers.py

```diff
@@ -0,0 +1,324 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
When moving the contents of modules around should be a seperate PR, it's very hard to check on what code is new and what already existed.